### ISSUE TITLE
a11y: Unclear functionality of input field - filter input

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13835]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**
a11y issue `59`

As soon as user starts entering data in mentioned input field, the content is automatically updated below in the table resulting in screen reader users unable to understand the dynamic update happening on the page.

Homepage
Account and API Keys
Projects/Namespaces
 - Main Content
  - Filter input field

Search
- Type to search for a resource input field (Available on activating  resource search button)

Provide a hidden instruction such as "On entering data into the following input field, the content will update below" before the input field on the page to orient screen reader users about dynamically updating content. Position this text off-screen via CSS.

**To Reproduce**

**Result**

**Expected Result**

When using a screen reader, users should understand through the instruction that on entering any data in the mentioned input field, the content below will be updated.

**Screenshots**

**Additional context**

**Comment by izaac** [comment]
Rancher version: `v2.12-49289cc9c6590b361d64950977dd20b1214908d7-head`
Rancher UI version (dashboard): `master d748be6`

"Using this input will immediately filter the results in the table below" is mentioned on the filter field element. 
This UI element is on all resource tables in the dashboard.

✅
